01_prepare_host.sh

```diff
@@ -96,9 +96,9 @@ source lib/releases.sh
 ## Install krew
 if ! kubectl krew > /dev/null 2>&1; then
   cd "$(mktemp -d)" &&
-  OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+  KERNEL_OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
```

Comment:
I wonder if we ever expect anything other than Linux here. Like, what?

Reply:
As discussed while back about supported OS, we're currently limiting ourselves to Ubuntu + Centos/RHEL already, so there would be no harm of hard coding it to Linux, but on the other hand, this doesn't hurt anyone either.